survey/SHACL_SurveyResultDataOffering.ttl

```diff
@@ -0,0 +1,52 @@
+@prefix sh: <http://www.w3.org/ns/shacl#> .
```

Comment:
In general:
ontology file is missing, example claims are missing.
See guidelines